docs/tools/vdb_table/data/pinecone.json

```diff
@@ -68,38 +68,38 @@
     "comment": ""
   },
   "embeddings_text": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/pinecone-io/canopy",
     "comment": ""
   },
   "embeddings_image": {
-    "support": "none",
-    "source_url": "",
+    "support": "",
+    "source_url": "https://github.com/pinecone-io/image-search-example",
     "comment": ""
   },
   "embeddings_structured": {
     "support": "",
-    "source_url": "",
+    "source_url": "https://github.com/pinecone-io/examples/blob/master/learn/search/question-answering/table-qa.ipynb",
```

Comment:
The structured data here refers to graphs:
https://github.com/superlinked/VectorHub/blob/7b6ab3883d400a12169b0942f21de4d51eb6eb82/docs/tools/vdb_table/vendor.schema.json#L24

The Table QA notebook example would not qualify here.

Reply:
oops, modified